gaj23/face-it
# Ch/feature/refactor

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

Refactor search and component and rendering for search if there are no matching results.  Cleaned up formatting and added hover and active states to images and links.  

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

Upon attempting to display "backup" images for broken image links, it was discovered that the images are valid links but the actual images are unavailable so error handling was created as an issue for a future iteration.

> Describe your code changes in detail for reviewers.

App.js was refactored in the return statement a well as in the SearchResults component.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

View css files to observe changes made to active and hover states.

App.js and SearchResults.js were modified.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Run `npm start` in your terminal to view the site on localhost:3000 as well as start the server by entering nodeserver.js in another terminal window.  Enter a search term and you should see a message pop up if there are no results or you will see your results displayed.

### Steps To Take
> Any questions and possible steps moving forward.

Every thing seems to be working as intended by this point.  We still need to clean up tags for accessibility and refactor some of our larger methods but we should be on track! 

#### Checklist:
